src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const property of action.keysToRemove) {
+          if (Object.keys(state).filter(elem => elem === property)) {
+            delete state[property];
+          }
```

Comment:
You don't need to check if this property exists, method delete do it by itself
```suggestion
        for (const property of action.keysToRemove) {
                    delete state[property];
        
```

You don't need to check if this property exists, method delete do it by itself

Reply:
100%!You are right, thanks